backend/dal/Helpers/Extensions/PropertyExtensions.cs

```diff
@@ -290,43 +290,50 @@ public static void ThrowIfPropertyInSppProject(this Entity.Property property, Cl
         }
 
         /// <summary>
-        /// Get the latest workflow associated to this property, using the workflow sort order.
+        /// Get the latest project associated to this property, using the workflow sort order.
         /// </summary>
         /// <param name="property"></param>
         /// <returns>The workflow code of the latest workflow associated to this property</returns>
-        public static String GetLatestWorkflowCode(this Entity.Property property)
+        public static Entity.Project GetLatestProject(this Entity.Property property)
```

Comment:
moved this to its own function, since it was being duplicated.

Reply:
updated this function so that it will not include any projects that are in a terminal status.